Framework/Algorithms/inc/MantidAlgorithms/PhaseQuadMuon.h

```diff
@@ -63,9 +63,11 @@ class DLLExport PhaseQuadMuon : public API::Algorithm {
   API::MatrixWorkspace_sptr squash(const API::MatrixWorkspace_sptr &ws,
                                    const API::ITableWorkspace_sptr &phase,
                                    const std::vector<double> &n0);
+  int findName(const std::vector<std::string> patterns,
```

Comment:
Do we need this to be a member function? It could be in an anonymous namespace? Also the patterns `vector` should be taken by reference.